From: Chuck Ebbert <cebbert@redhat.com>
Subject: ata: sata_nv SWNCQ should be disabled by default
Reverts: d21279f4125893c63ec285962e1f2164b4d71117

libata: sata_nv: SWNCQ should be disabled by default

https://bugzilla.redhat.com/show_bug.cgi?id=463034

Signed-off-by: Chuck Ebbert <cebbert@redhat.com>
---

diff --git a/drivers/ata/sata_nv.c b/drivers/ata/sata_nv.c
index b2eb572..8bb8ba1 100644
--- a/drivers/ata/sata_nv.c
+++ b/drivers/ata/sata_nv.c
@@ -532,7 +532,7 @@ MODULE_DEVICE_TABLE(pci, nv_pci_tbl);
 MODULE_VERSION(DRV_VERSION);
 
 static int adma_enabled;
-static int swncq_enabled = 1;
+static int swncq_enabled;
 
 static void nv_adma_register_mode(struct ata_port *ap)
 {
@@ -2485,7 +2485,7 @@ module_exit(nv_exit);
 module_init(nv_init);
 module_exit(nv_exit);
 module_param_named(adma, adma_enabled, bool, 0444);
-MODULE_PARM_DESC(adma, "Enable use of ADMA (Default: true)");
+MODULE_PARM_DESC(adma, "Enable use of ADMA (Default: false)");
 module_param_named(swncq, swncq_enabled, bool, 0444);
-MODULE_PARM_DESC(swncq, "Enable use of SWNCQ (Default: true)");
+MODULE_PARM_DESC(swncq, "Enable use of SWNCQ (Default: false)");
 
